externs/browser/w3c_audio.js

```diff
@@ -26,245 +26,305 @@
 /**
  * @constructor
  */
-function AudioContext() {}
+function BaseAudioContext() {};
 
-/** @type {!AudioDestinationNode} */
-AudioContext.prototype.destination;
+/**
+ * @type {number}
+ * @const
```

Comment:
The compiler automatically recognizes a type given after `@const`, so this can just be.

```javascript
/** @const {number} */
```